pandas/tests/groupby/test_groupby.py

```diff
@@ -1490,6 +1490,19 @@ def test_frame_describe_multikey(self):
         for name, group in groupedT:
             assert_frame_equal(result[name], group.describe())
 
+    # GH #14848
+    def test_frame_describe_tupleindex(self):
+        df1 = DataFrame({'x': [1, 2, 3, 4, 5] * 3,
+                         'y': [10, 20, 30, 40, 50] * 3,
+                         'z': [100, 200, 300, 400, 500] * 3})
+        df1['k'] = [(0, 0, 1), (0, 1, 0), (1, 0, 0)] * 5
+        df2 = df1.rename(columns={'k': 'key'})
+        des1 = df1.groupby('k').describe()
+        des2 = df2.groupby('key').describe()
+        if len(des1) > 0:
```

Comment:
I wasn't sure the index would exist if the `DataFrame` was empty.  Retested and see that I don't need the test.